### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Screen Readers][MAS1.3.1][HighImpact][Shortcut key associated with command is not announced by Screen reader in Report generation Frame.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#712]
**User Experience:**
User who rely on AT will not get information about the shortcut keys , if shortcut key is not announced by screen reader.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Narrator
**Attachments**:
[MAS1.3.1_Shortcut key is not announced.zip](https://github.com/OfficeDev/script-lab/files/1457429/MAS1.3.1_Shortcut.key.is.not.announced.zip)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to “report generation” Iframe and Turn on Narrator.
3. Press F1 key to open the picker and verify whether screen reader is announcing the short cut key command.

**Current Behavior:**
Screen reader is not announcing the short cut key command provided in the picker with the control name associated with it.

**Expected Behavior:**
Screen reader  should announce the shortcut key command along with control which helps AT user to perform the action. 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.3.1
